data/events.yaml

```diff
@@ -198,3 +198,39 @@ x-events:
     cluster_id: UUID
     failed_operators: string
 
+<<<<<<< HEAD
+=======
+=======
+    cluster_name: string
+    failed_operators: string
+<<<<<<< HEAD
+>>>>>>> 95826a5e... MGMT-4872: Generate cluster update events
+=======
+
+- name: download_image_failed_fetch
+  message: "Failed to download image: error fetching from storage backend"
+  event_type: infra_env
+  severity: "error"
+  properties:
+    infra_env_id: UUID
+
+- name: host_deregistered_cluster
```

Comment:
We should also have event of host deregistering from Infraenv.  Or may we can just have host derigster event, and and will have an optional cluster id parameter. @avishayt ?

Reply:
I would go with the second option - host deregister event with optional cluster ID

Reply:
I see that the `host_deregistered_cluster` event got merged in https://github.com/openshift/assisted-service/pull/2497, so we should probably open a ticket to follow up with that.
@masayag